src/vm/exceptionhandling.cpp

```diff
@@ -4663,6 +4664,11 @@ VOID DECLSPEC_NORETURN DispatchManagedException(PAL_SEHException& ex)
                     // Unwind to the managed caller of the helper
                     PAL_VirtualUnwind(&frameContext, NULL);
                 }
+#ifdef _TARGET_ARM_
+                if (IsDispatchStub(&frameContext)) { // access violation on DispatchStub
+                    AdjustContextForVirtualStub(&ex.ExceptionRecord, &frameContext);
```

Comment:
Have you verified that this actually works? I remember that when we were fixing the problem with IsIPInMarkedJitHelper, @myungjoo said that the context of the hardware exception didn't have the LR set to the caller's PC and so the simple attempt to unwind by setting PC to LR as the AdjustContextForVirtualStub also does didn't work and we had to use the PAL_VirtualUnwind instead.